src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -34,6 +33,10 @@
 #include <app/reporting/reporting.h>
 #include <app/server/CommissioningWindowManager.h>
 #include <app/server/Server.h>
+#include <clusters/GeneralCommissioning/Attributes.h>
```

Comment:
metadata is sufficient I believe.

Should we add dependency in the BUILD.gn group to not get it transitively?

Reply:
I removed the cluster-objects.h.